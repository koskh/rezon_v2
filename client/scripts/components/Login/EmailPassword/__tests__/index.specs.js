import * as React from 'react';

import {shallow, mount} from 'enzyme';
import renderer from 'react-test-renderer';

import EmailPassword from '../';

describe('components/DumbBtn', () => {

    test('Component can render', () => {
        const component = renderer.create(<EmailPassword title='TestTitle'/>);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    test('Component change title on click', () => {

        const testTitle= 'TestTitle';
        const component = mount(<EmailPassword title={testTitle}/>);

        expect(component.text()).toEqual(testTitle);
        component.simulate('click');
        expect(component.text()).toEqual('Bye');
    });

});
