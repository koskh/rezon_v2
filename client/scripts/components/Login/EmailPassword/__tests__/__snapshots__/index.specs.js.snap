// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/DumbBtn Component can render 1`] = `
<button
  onClick={[Function]}
>
  TestTitle
</button>
`;
